google-map.html

```diff
@@ -480,10 +480,7 @@
       if (this._mutationObserver) {
         return;
       }
-      this._mutationObserver = new MutationObserver(this._updateMarkers.bind(this));
-      this._mutationObserver.observe(this.$.selector, {
-        childList: true
-      });
+      this._mutationObserver = this.listen(this.$.selector, 'items-changed', '_updateMarkers');
```

Comment:
let's rename this property to something more appropriate: `this._markersChildrenListener`